modules/core_project_factory/main.tf

```diff
@@ -193,64 +206,50 @@ data "null_data_source" "default_service_account" {
   }
 }
 
-/******************************************
-  Default compute service account deletion
- *****************************************/
-resource "null_resource" "delete_default_compute_service_account" {
-  count = var.default_service_account == "delete" ? 1 : 0
+/************************************************************************************
+  Default compute service account action. Options are keep, depriviledge, delete.
+ ***********************************************************************************/
+resource "null_resource" "modify_default_compute_service_account" {
```

Comment:
Renaming the resource is likely to lead to some issues with existing projects (causing state migrations). I'd prefer if we kept these as two separate null_resources.

Reply:
Cool make sense, this has been fixed in the latest commit.